src/App.js

```diff
@@ -1,10 +1,11 @@
 import './App.css';
-import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
```

Comment:
- [ ] Hi, Can you remove this commented code to make avoid confusion? Believe me, clean code will make your code more attractive, and disciplined in the eyes of the recruiters.